### PR TITLE
Update centralized-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the wording to reflect that one needs to download and install the compatibility checker

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9877

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ In order to deploy an add-in via Centralized Deployment, you need to be either a
 
 ### Centralized Deployment Compatibility Checker
 
-Using the Centralized Deployment Compatibility Checker, you can verify whether the users on your tenant are set up to use Centralized Deployment for Word, Excel and PowerPoint. The Compatibility Checker is not required for Outlook support. Download the [compatibility checker](https://aka.ms/officeaddindeploymentorgcompatibilitychecker).
+Using the Centralized Deployment Compatibility Checker, you can verify whether the users on your tenant are set up to use Centralized Deployment for Word, Excel and PowerPoint. The Compatibility Checker is not required for Outlook support. Download and install the [compatibility checker](https://aka.ms/officeaddindeploymentorgcompatibilitychecker).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
Using the Centralized Deployment Compatibility Checker, you can verify whether the users on your tenant are set up to use Centralized Deployment for Word, Excel, and PowerPoint. The Compatibility Checker is not required for Outlook support. Download and install the [compatibility checker](https://aka.ms/officeaddindeploymentorgcompatibilitychecker).
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba copy review

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Pull request merged**